tests/commetsCounter.test.js

```diff
@@ -0,0 +1,18 @@
+const loadComments = require('../src/modules/commentsLoader.js');
+
+const comments = [
+  { creation_date: '2021/1/5', username: 'John', comment: 'Wow I love it!' },
+  { creation_date: '2022/3/8', username: 'Mike', comment: 'Yummy!' },
+];
+
+describe('Comments test', () => {
+  it('Should insert and count the comments', () => {
+    document.body.innerHTML = `
+      <h3 class="mt-3 text-lg"><span id="modal-comments-count" class="font-bold"></span> comments</h3>
+      <ul id="modal-comments"></ul>
+    `;
+    loadComments(comments);
+    const commentsCount = document.querySelector('#modal-comments-count').textContent;
+    expect(comments.length).toBe(Number(commentsCount));
```

Comment:
In the general case of an equality (i.e. `a = b`)  `a` and `b` can be interchanged according to the _Commutative Property_. But for the test to be clear, it would be more appropriate that you expect the _result of your test_ `toBe` _a well-known value._ Please consider interchanging the `Number(commentsCount)` to the `expect()` and `comments.length` to the `toBe()`.

Have you considered what would happened in case `comments = []`? That could be another interesting case to test.

Reply:
I'm now using `toEqual` for the test, and I also added another test to check for the empty array.